src/Platforms/SQLitePlatform.php

```diff
@@ -905,7 +917,11 @@ private function getForeignKeysInAlteredTable(TableDiff $diff): array
                     ->getValue(),
             );
 
-            unset($foreignKeys[$constraintKey]);
+            if (! isset($keysByName[$constraintKey])) {
+                continue;
+            }
```

Comment:
By assertion I mean `assert(isset($keysByName[$constraintKey]))`. This way, we are saying that we always expect it to be set, otherwise it should be an error.

Making it a `continue` sends the wrong message and adds code not covered by tests.